pkg/e2etemplates/cert_operator_chart_values.go

```diff
@@ -25,4 +25,5 @@ Installation:
       Registry:
         PullSecret:
           DockerConfigJSON: "{\"auths\":{\"quay.io\":{\"auth\":\"$REGISTRY_PULL_SECRET\"}}}"
+labelSelector: 'giantswarm.io/cluster=${CLUSTER_NAME}'
```

Comment:
Can we in the next step make sure we are not using the env vars anymore?

Reply:
Probably. Could take a bit of time as it's a breaking change to quite many places.

Reply:
Oh yeah but this is broken right now anyway.

Reply:
At least it's used above for `clusterName` value.